packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap

```diff
@@ -0,0 +1,2056 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CSS snapshots variables in cmsgov-theme.css match snapshot 1`] = `
```

Comment:
Note that these represent the variables generated by the old token system. I just want to check this into our feature branch so we have something to compare against as we address the CSS changes

Reply:
There's a comment in the test module that generates it, but it's not marked as a TODO. I just commented that it can be deleted as soon as it's causing more trouble than it's worth